chia/wallet/wallet_state_manager.py

```diff
@@ -1657,7 +1702,6 @@ async def create_wallet_for_puzzle_info(self, puzzle_driver: PuzzleInfo, name=No
 
     async def add_new_wallet(self, wallet: WalletProtocol) -> None:
         self.wallets[wallet.id()] = wallet
-        await self.create_more_puzzle_hashes()
```

Comment:
This shouldn't be removed, the method still needs to be called. The comment was only about the new parameter to skip calling this method.

Reply:
whoops, misunderstood the diff, fixing.